app/scripts/controllers/client/ViewClientController.js

```diff
@@ -30,6 +32,21 @@
                     location.path('/viewrecurringdepositaccount/' + id);
                 }
             };
+
+            scope.chargesPerPage = 200;
+
+            scope.initPage = function () {
+
+                var chargesItems = resourceFactory.clientChargesResource.getCharges({clientId: routeParams.id,
```

Comment:
we only want charges which are not paid. The original implementation used to pass an argument paid=false, we should do the same here